upup/models/cloudup/resources/addons/node-termination-handler.aws/k8s-1.11.yaml.template

```diff
@@ -183,6 +197,8 @@ spec:
               value: "false"
             - name: ENABLE_SQS_TERMINATION_DRAINING
               value: "true"
+            - name: EXCLUDE_FROM_LOAD_BALANCERS
+              value: "{{ .ExcludeFromLoadBalancers }}"
```

Comment:
You need to use `WithDefaultBool` here since `ExcludeFromLoadBalancers` is a pointer and go templates will evaluate pointers as non-nil or not, also for boolean pointers.

Reply:
Okay, I will try to use WithDefaultBool for all nth manifest.